charts/elasticsearch-umbrella/charts/elasticsearch-deployment/values.yaml

```diff
@@ -10,11 +10,11 @@ replicaCount: 3
 
 image:
   # -- Docker repository for Elasticsearch image
-  repository: eu.gcr.io/carrefour-fr-750c11a47fecaeff/empathy-search/elasticsearch-gcp
+  repository: harbor.internal.shared.empathy.co/empathyco/elasticsearch
```

Comment:
There's a conversation open about where we should store this image. I think that using Harbor is not the right one but we would need then (given it's a public repo) to publish it in Dockerhub, what do you think? Elastic public image is not working because of the memlock

Reply:
Resolved. @Roberdvs manually replicated it to dockerhub. Thanks Rober!